modules/saml/src/Controller/ServiceProvider.php

```diff
@@ -165,6 +165,14 @@ protected function loginHandler(
         ) {
             throw new Error\BadRequest('Missing ReturnTo parameter.');
         }
+        if (
+            $request->query->has('ReturnTo') &&
```

Comment:
I think this entire block can be replaced with `$request->query->getString('ReturnTo') === ''`.
If the parameter doesn't exist, the function will return `''`

Reply:
Indeed, much simpler

Reply:
Yeah, but it does need to exist, but be empty for this exception to trigger. The rest is handled above and is more complex (it's ok if it's completely missing)

Reply:
I've simplified the code a bit with getString()